### PR TITLE
Implement job cancellation in UI

### LINKED ISSUE #350: Add job cancel in the UI
**Is your feature request related to a problem or challenge? Please describe what you are trying to do.**
We should add the job cancelling in te UI

**Describe the solution you'd like**
Implement it

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Dandandan**
# Which issue does this PR close?

Closes #350

 # Rationale for this change
Implementing job cancellation in UI and some small cleanup / fixing:

image

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

File: ballista/ui/scheduler/src/components/QueriesList.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export enum QueryStatus {
 export interface Query {
   job_id: string;
   job_name: string;
-  status: QueryStatus;
+  job_status: string;
```
Comment (Dandandan) [inline ballista/ui/scheduler/src/components/QueriesList.tsx:54]: Fix - current API doesn't return `status` field.

File: ballista/ui/scheduler/src/components/QueriesList.tsx [hunk 2]
```diff
@@ -94,13 +94,20 @@ export const ActionsCell: (props: any) => React.ReactNode = (props: any) => {
   };
   return (
     <Flex>
-      <FaStop color={"red"} title={"Stop this job"} />
+      <FaStop color={"red"} title={"Stop this job"} onClick={() => {
+        fetch("api/job/" + props.value.job_id, {
```
Comment (Dandandan) [inline ballista/ui/scheduler/src/components/QueriesList.tsx:98]: This works succesfully - but has the "infinite loop" problem
https://github.com/apache/arrow-ballista/issues/341

File: ballista/ui/scheduler/src/components/Header.tsx [hunk 3]
```diff
@@ -53,15 +53,6 @@ export const Header: React.FunctionComponent<HeaderProps> = ({
         <NavBarContainer>
           <Flex flexDirection={"row"} alignItems={"center"}>
             <img alt={"Ballista Logo"} src={Logo} />
-            <Text
```
Comment (Dandandan) [inline ballista/ui/scheduler/src/components/Header.tsx:56]: Seems a bit redundant, I think we can remove it here

File: ballista/ui/scheduler/src/components/Summary.tsx [hunk 4]
```diff
@@ -64,7 +64,7 @@ export const Summary: React.FunctionComponent<SummaryProps> = ({
             fontWeight={"normal"}
           >
             <Text fontWeight={"light"} mb={2}>
-              General Cluster Info
+              Cluster Details
```
Comment (Dandandan) [inline ballista/ui/scheduler/src/components/Summary.tsx:67]: Not sure if text is necessary, but "Cluster Details" seems a bit more appropriate.

**Review by andygrove — APPROVED** [review]
Thanks @Dandandan. I tested this locally and I could cancel a job from the UI.

I did notice a glitch though where the job shows up twice on the UI. I filed https://github.com/apache/arrow-ballista/issues/355 to track this

**Pull request merged**